server/testdriver/core/src/main/java/org/infinispan/server/test/api/HotRodTestClientDriver.java

```diff
@@ -70,27 +70,13 @@ public HotRodTestClientDriver withPort(int port) {
    }
 
    /**
-    * Created a cache with the name of the method where this method is being called from.
-    * If the cache already exists, retrieves the existing cache
-    *
-    * @return {@link RemoteCache}, the cache
-    */
-   public <K, V> RemoteCache<K, V> create() {
-      return create(-1);
-   }
-
-   /**
-    * Create a cache adding in the initial server list the server address given by the index
-    * @param index the server index, -1 for all
+    * Create a cache adding in the initial server list the server address given by the index.
+    * If empty, it will add all servers by default OR those that were configured in the server list
+    * @param index the server index
     * @return {@link RemoteCache}, the cache
     */
-   public <K, V> RemoteCache<K, V> create(int index) {
-      RemoteCacheManager remoteCacheManager;
-      if (index >= 0) {
-         remoteCacheManager = createRemoteCacheManager(index);
-      } else {
-         remoteCacheManager = createRemoteCacheManager();
-      }
+   public <K, V> RemoteCache<K, V> create(int... index) {
```

Comment:
I liked it because now we have only one method instead of two.
Also, it allows having more than one server in the client initial server list configuration. The previous version of the method allows only one.